src/lib/api/observations.js

```diff
@@ -1,5 +1,26 @@
 import convert from "https://cdn.jsdelivr.net/npm/convert@4";
 
+// equatorial mean radius of Earth (in miles)
+const R = 3_963.19;
+
+function toRad(x) {
+  return (x * Math.PI) / 180.0;
+}
+function hav(x) {
```

Comment:
is there a style guide? if so, maybe the brackets should be removed if there's only one thing being removed